src/logger/console.ts

```diff
@@ -33,7 +33,7 @@ export default class ConsoleLogger implements Logger {
 	 * @returns The formatted message with context
 	 * @private
 	 */
-	private formatMessage(message: string, args: unknown[]): string {
+	private formatMessage(message: string | number, args: unknown[]): string {
```

Comment:
i think probably we just need to accept any object and then if not a scalar (number, string, boolean) we just json.stringify or use the util.inspect to create a nice formatted object